src/init/adduser.sh

```diff
@@ -79,6 +83,8 @@ else
             GID=$(cat /etc/group | grep wazuh| cut -d':' -f 3)
             uid=$(( $GID + 1 ))
             echo "${USER}:x:$uid:$GID::${DIR}:/bin/false" >> /etc/passwd
+        elif [ $(grep "Alpine Linux" /etc/os-release > /dev/null  && echo 1) ]; then
+            ${USERADD} "${USER}" -H "${DIR}" -s ${OSMYSHELL} -G "${GROUP}"
```

Comment:
Regarding the help message of `adduser` in Alpine Linux:
```
BusyBox v1.35.0 (2022-08-01 15:14:44 UTC) multi-call binary.

Usage: adduser [OPTIONS] USER [GROUP]

Create new user, or add USER to GROUP

	-h DIR		Home directory
	-g GECOS	GECOS field
	-s SHELL	Login shell
	-G GRP		Group
	-S		Create a system user
	-D		Don't assign a password
	-H		Don't create home directory
	-u UID		User id
	-k SKEL		Skeleton directory (/etc/skel)
```

You have to use `-h` instead of `-H` for the home directory. Otherwise, it will create a default one:
```
wazuh:x:100:101:Linux User,,,:/home/wazuh:/sbin/nologin
```

Apart from that, it works as expected.

Reply:
I have changed that option to have the same behavior as other OS!